server/fleet/scripts.go

```diff
@@ -571,10 +571,14 @@ var (
 )
 
 type BatchExecutionSummary struct {
-	ScriptID   uint                 `json:"script_id" db:"script_id"`
-	ScriptName string               `json:"script_name" db:"script_name"`
-	TeamID     *uint                `json:"team_id" db:"team_id"`
-	Hosts      []BatchExecutionHost `json:"hosts"`
+	ScriptID    uint   `json:"script_id" db:"script_id"`
+	ScriptName  string `json:"script_name" db:"script_name"`
+	TeamID      *uint  `json:"team_id" db:"team_id"`
```

Comment:
This is the only nullable field here right? What will `team_id: null` indicate?

Reply:
`null` team_id represents a script on "No Team"

Reply:
And 0 would indicate all teams, if this was supported for all teams, which it's not?

Reply:
(some reconciliation is needed a "No team" is team 0 in the UI)

Reply:
If it'd be better to have 0 represent "no team" I can make that change

Reply:
Ah, okay, will update FE

Reply:
Doesn't matter, easy translation

Reply:
Done